jobs/delorean/2.0/ews/3scale/discovery.yaml

```diff
@@ -22,6 +22,14 @@
     - string:
           name: CI_MESSAGE
```

Comment:
@psturc Are you using uppercase variable names in the new jenkins configuration repo? Is that something that was decided upon?

Reply:
@mikenairn in the end we decided to go with camelCase: https://gitlab.cee.redhat.com/integreatly-qe/ci-cd/blob/master/jobs/Integreatly/rhmi-install-addon-flow.groovy